infra/README.md

```diff
@@ -11,3 +11,14 @@ After deployment you should add security groups for wireguard because by default
 Set origins in CORS headers manually, because tf does not support that
 
     vault write sys/config/cors allowed_origins="*"
+
+
+## Registry
+
+Deploys 3 instances and set up docker registries on them.
+
+Takes your ssh public key as input
+
+```bash
+sudo terraform init && sudo terraform apply --auto-approve && cd docker-registry/ && sleep 25 && sudo ansible-playbook deploy-docker-registry.yaml
+```
```

Comment:
nice readme!

We add pubkeys to https://github.com/mekstack/mekstack/blob/master/infra/admin/authorized_keys and pass the keypair name to the terraform module. Please do the same